sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -49,8 +49,13 @@ object DateTimeUtils {
   final val TimeZoneUTC = TimeZone.getTimeZone("UTC")
 
   val TIMEZONE_OPTION = "timeZone"
+  val TIME_ZONE_ID = "(\\+|\\-)(\\d)(:)"
 
-  def getZoneId(timeZoneId: String): ZoneId = ZoneId.of(timeZoneId, ZoneId.SHORT_IDS)
+  def getZoneId(timeZoneId: String): ZoneId = {
+    // To support (+|-)h:mm ZoneOffset format
+    // Because (+|-)h:mm string is unsupported format in 3.0
```

Comment:
So, the actual reason is it was supported before 3.0. Could you change it to:
```
To support the (+|-)h:mm format because it was supported before Spark 3.0.
```